### PR TITLE
CTCP-5949: Added hint text to optional pages

### PR DESCRIPTION
**Pull request by kevincoatsworth**

### CODE REVIEW

**Comment by kevincoatsworth** [comment]
Items - add payment method yes no
Items - add additional information yes no
Items - add additional reference number yes no
Items - add additional reference yes no
Items - add supply chain actor yes no
Items - add supplementary units yes no
Items - add net weight yes no
Items - add shipping mark yes no
Items - add combined nomenclature code yes no
Items - add commodity code yes no
Items - add cus code yes no
Items - add ucr yes no

File: app/views/item/AddAdditionalInformationYesNoView.scala.html [hunk 1]
```diff
@@ -45,7 +45,8 @@
                 heading = messages("item.addAdditionalInformationYesNo.heading"),
                 caption = Some(messages("item.additionalInformation.caption", itemIndex.display)),
                 additionalHtml = messages("item.addAdditionalInformationYesNo.paragraph").toParagraph
-            )
+            ),
+            hint = Some(HtmlContent(messages("item.addAdditionalInformationYesNo.hint")))
```
Comment (jackheslop96) [inline app/views/item/AddAdditionalInformationYesNoView.scala.html:49]: `HtmlContent` is designed for raw html. Use `Text` as the content type when the input is a string

Comment (kevincoatsworth) [reply app/views/item/AddAdditionalInformationYesNoView.scala.html:49]: I followed suit with an existing hint in the repo, should I change those too?

Comment (jackheslop96) [reply app/views/item/AddAdditionalInformationYesNoView.scala.html:49]: Yes please 👍

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**